src/components/pages/rules/rulesGrid/rulesGrid.js

```diff
@@ -4,7 +4,7 @@ import React, { Component } from 'react';
 import { translate } from 'react-i18next';
 import { PcsGrid } from 'components/shared';
 import { rulesColumnDefs, checkboxParams, defaultRulesGridProps } from './rulesGridConfig';
-import { isFunc } from 'utilities';
+import { isFunc, translateColumnDefs } from 'utilities'; //translateColumnDefs
```

Comment:
The comment at the end of the line should be deleted.

Reply:
Deleted